test/src/test/java/hudson/cli/AbstractBuildRangeCommand2Test.java

```diff
@@ -52,7 +52,7 @@
 /**
  * @author pjanouse
  */
-public class AbstractBuildRangeCommandTest2 {
+public class AbstractBuildRangeCommand2Test {
```

Comment:
Introduced in https://github.com/jenkinsci/jenkins/pull/2257, along with `AbstractBuildRangeCommandTest`. Not obvious to me why both exist—perhaps because one uses `@Rule` and the other `@ClassRule`?